src/ripple/rpc/handlers/AccountChannels.cpp

```diff
@@ -40,9 +40,12 @@ void addChannel (Json::Value& jsonLines, SLE const& line)
     jDst[jss::destination_account] = to_string (line[sfDestination]);
     jDst[jss::amount] = line[sfAmount].getText ();
     jDst[jss::balance] = line[sfBalance].getText ();
-    PublicKey const pk (line[sfPublicKey]);
-    jDst[jss::public_key] = toBase58 (TokenType::TOKEN_ACCOUNT_PUBLIC, pk);
-    jDst[jss::public_key_hex] = strHex (pk);
+    if (publicKeyType(line[sfPublicKey]))
+    {
+        PublicKey const pk (line[sfPublicKey]);
+        jDst[jss::public_key] = toBase58 (TokenType::TOKEN_ACCOUNT_PUBLIC, pk);
```

Comment:
I look into removing this. The `channel_verify` command takes the public key as base58 encoded. It's convenient to get that value from this command without having to convert. 

Of course, I _could_ change the `channel_verify` command to take the public key as hex, but I don't want to hold up this PR. I'd like to keep the encoding.